pyfar/classes/transmission_matrix.py

```diff
@@ -739,6 +740,108 @@ def create_gyrator(transducer_constant :
         frequencies = transducer_constant.frequencies
         return TransmissionMatrix.from_abcd(0, B, C, 0, frequencies)
 
+    @staticmethod
+    def create_transmission_line(
+            wavenumber: FrequencyData,
+            length: float | FrequencyData,
+            characteristic_impedance: complex | FrequencyData,
```

Comment:
I'm assuming this can also be a float?
Type hint and docstring are not matching for length and impedance.

Reply:
I introduced `Number` for the characteristic impedance both in type hint and docstring